app/src/lib/git/reflog.ts

```diff
@@ -63,3 +63,41 @@ export async function getRecentBranches(
 
   return [...names]
 }
+
+/**
+ * Gets the distinct list of branches that have been checked out after a specific date
+ */
+export async function getCheckoutsAfterDate(
+  repository: Repository,
+  afterDate: Date
+): Promise<ReadonlyArray<string>> {
+  //regexr.com/46n1v
+  const regex = new RegExp(
+    /^(?:.{9})\s(?:\(.*\))?\s?(?:HEAD@){(.*)}:\s(?:.*)to\s(.*)$/
+  )
+  const isoDate = afterDate.toISOString()
+  const filterDate = isoDate.substr(0, isoDate.indexOf('T'))
```

Comment:
Can we make it clear somewhere that this filter date is omitting the hour/month/second value from whatever it receives, and why this is important?

Reply:
I actually thought that was the format `iso` needed. Now that I know that's not the case, I'm now passing `afterDate.toIsoString()` instead of building a date string by omitting the time information.